schema-engine/sql-introspection-tests/tests/mysql/constraints.rs

```diff
@@ -0,0 +1,101 @@
+//! https://www.notion.so/prismaio/Indexes-Constraints-Check-constraints-MySQL-e56a210937904cae91836e202830202a
+
+use indoc::indoc;
+use sql_introspection_tests::test_api::*;
+use test_macros::test_connector;
+
+// Note: MySQL 5.6 ad 5.7 do not support check constraints, so this test is only run on MySQL 8.0.
+#[test_connector(tags(Mysql8), exclude(Vitess, Mariadb))]
```

Comment:
The `exclude(Mariadb)` here is not necessary, if we're only running the test on MySQL 8. Also, why skip vitess?